README.md

```diff
@@ -86,7 +86,7 @@ multiselectPlugin.init(options);
 - `inMultipleSelectionModeWeakMap`: The WeakMap storing whether the plugin is in multiple selection mode by workspace svg.
 
 ## Credit
-- [ds.min.js](lib/ds.min.js): This plugin uses [a patched version of DragSelect](https://github.com/ThibaultJanBeyer/DragSelect/pull/128) to realize "drag a rectangle to select multiple blocks" feature.
+- [DragSelect]([lib/ds.min.js](https://github.com/ThibaultJanBeyer/DragSelect)): This plugin uses [a patched version of DragSelect](https://github.com/ThibaultJanBeyer/DragSelect/pull/143) to realize "drag a rectangle to select multiple blocks" feature (has got merged into v2.4.4).
```

Comment:
It looks like you are no longer using your own patched version, so you can remove references to `lib/ds.min.js` and can remove the references to the patched version and the pull request.

Reply:
Sorry about this, it was just some stupid VSCode auto-completion that I didn't notice...